src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs

```diff
@@ -92,6 +92,35 @@ public MvcOptions()
         /// </summary>
         public FormatterCollection<IInputFormatter> InputFormatters { get; }
 
+        /// <summary>
+        /// Gets or sets an value indicating whether the model binding system will bind undefined values to 
+        /// enum types. The default value of the property is <c>false</c>. 
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// This property is associated with a compatibility switch and can provide a different behavior depending on 
+        /// the configured compatibility version for the application. See <see cref="CompatibilityVersion"/> for 
+        /// guidance and examples of setting the application's compatibility version.
+        /// </para>
+        /// <para>
+        /// Configuring the desired of the value compatibility switch by calling this property's setter will take precedence
+        /// over the value implied by the application's <see cref="CompatibilityVersion"/>.
+        /// </para>
+        /// <para>
+        /// If the application's compatibility version is set to <see cref="CompatibilityVersion.Version_2_0"/> then
+        /// this setting will have value <c>false</c> if not explicitly configured.
+        /// </para>
+        /// <para>
+        /// If the application's compatibility version is set to <see cref="CompatibilityVersion.Version_2_1"/> or
+        /// higher then this setting will have value <c>true</c> if not explicitly configured.
+        /// </para>
+        /// </remarks>
+        public bool SuppressBindingUndefinedValueToEnumType
```

Comment:
@Eilon for docsfun. My intent is create a good copy-pastable blurb we can reuse and tweak for each setting.

Reply:
Yeah I think this is good.